src/DonationForms/resources/styles/design-settings/image.scss

```diff
@@ -216,3 +218,28 @@
     }
 }
 
+/* Two-Panel */
+.givewp-donation-form > .givewp-layouts-header {
+    .givewp-layouts-header__templates-ms {
```

Comment:
@JoshuaHungDinh head up, we'll want to circle back to the default header template classnames so they are not multi-select specific as referring to here `givewp-layouts-header__templates-ms` since right now if you were to create a _new_ form design that is _not_ multi-step it would still have this classname.

We don't have to solve that here, just wanted to bring that up before releasing the header image and I can help do that part

Reply:
Yea, I'd like a quick chat about this when you get back.